app/carbonzipper/app.go

```diff
@@ -199,10 +199,12 @@ func initBackends(config cfg.Zipper, logger *zap.Logger) ([]backend.Backend, err
 		}).DialContext,
 	}
 
-	backends := make([]backend.Backend, 0, len(config.Backends))
-	for _, host := range config.Backends {
+	backends := make([]backend.Backend, 0, len(config.GetBackends()))
```

Comment:
The result of this can be cached to a variable, instead of being inline, and used in the next statement.

Reply:
+1 AFAIU we are talking about `GetBackends()`.

Reply:
I have updated and committed the change.